packages/@coorpacademy-components/src/atom/chip/index.js

```diff
@@ -0,0 +1,77 @@
+import React, {useMemo, useState, useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import get from 'lodash/fp/get';
+import Color from 'colorjs.io';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Provider from '../provider';
+import style from './style.css';
+
+const DELTA_LUMINOSITY = 0.08;
+const DEFAULT_BACKGROUND_COLOR = '#0061FF';
+const ICON_SIZE = '12px';
+
+const Chip = (props, context) => {
+  const {text, selected = false, onClick, backgroundColor = DEFAULT_BACKGROUND_COLOR} = props;
+  const {skin} = context;
+  const skinColor = get('common.primary', skin);
+  const selectedBgColor = backgroundColor === 'skin' && skinColor ? skinColor : backgroundColor;
+  const hoveredSelectedBgColor = new Color(selectedBgColor)
+    .to('hsl')
+    .set({l: l => l * (1 - DELTA_LUMINOSITY)})
+    .toString();
```

Comment:
extract function. (and quick test coverage)
(luminosityDelta can be a second arg, with `LUMINOSITY_DELTA` as default value)